sqld/src/query_analysis.rs

```diff
@@ -96,7 +96,7 @@ impl Statement {
                 kind,
             })
         }
-        let mut parser = Box::new(Parser::new(s.as_bytes()));
+        let mut parser = Parser::new(s.as_bytes());
```

Comment:
why did you do that?
The parser needs to be boxed because it's very big. please undo that, and I'll add a comment